main.go

```diff
@@ -32,14 +35,25 @@ func main() {
 		fmt.Println(version.Tag)
 		return
 	case "go":
-		err := goproxy.Run(args, goproxy.WithForceBuild(), goproxy.WithDifferentCache())
+		orchestrion, err := os.Executable()
 		if err != nil {
+			orchestrion = os.Args[0]
```

Comment:
How about using `filepath.Abs` in that case? If orchestrion is called from a relative path it may cause issues combined with toolexec down the road

Reply:
`os.Executable()` kind of does that already, but in a different way (and TBH it makes me a little uneasy when I read that code... but I suppose it was written by people smarter than me).

It nevertheless is a good idea. That should be an _extremely_ cold path in normal circumstances anyway!